### DISCUSSION TITLE
Helpdesk software as a new service

### DISCUSSION THREAD

**Discussion by isakgustavsen** [coollabsio/coolify#4609] [Service Template Requests]
Hello

Has there been any thoughts around helpdesk/ticketing software as a service? Having looked at a few most require an external db, but maybe that can be auto created to? 

My suggestion would be [UV Desk](https://www.uvdesk.com/en/), but there are a lot of them, and I do not know which ones would be best/easiest to integrate.

**Comment by kekeil** [comment]
Hello  @isakgustavsen,
You can easily deploy UVDesk in just a few minutes using Docker Compose. use this tutorial to do that on coolify easily https://wiki.opensourceisawesome.com/books/uvdesk-helpdesk-tracking-software/page/install-using-docker-and-docker-compose
